app/controllers/catalog_controller.rb

```diff
@@ -111,7 +111,7 @@ def atomic_st_edit
         @edit[:new] ||= {}
         @edit[:current] ||= {}
         @edit[:key] = "prov_edit__new"
-        @edit[:st_prov_types] = catalog_item_types
```

Comment:
We should not be putting constants in the session. I cannot imagine a reason to do this.